### DISCUSSION TITLE
pyvista vtk import problem

### DISCUSSION THREAD

**Discussion by earcmag** [pyvista/pyvista#3382] [Q&A] [answered]
Hi! Whenever I use pyvista functions that require an import from vtk I get the following: ModuleNotFoundError: No module named 'vtkIOParallelPython'. Yet I can see in the vtk ProgramData folder that there is a vtkIOParallel.py module. I have tried creating a new environment and installing packages (see below) from scratch. Running Python 3.7.6 and conda 22.9.0 in Windows. New to python so apologies if this is an easy mistake or I haven't provided enough information.

[conda_packages.txt](https://github.com/pyvista/pyvista/files/9675844/conda_packages.txt)

**Comment by banesullivan** [comment, marked as answer]
This is an odd issue, so please don't be discouraged! 

I have a feeling that `conda` is pulling VTK from the wrong channel for you.

In your conda list:

```
pyvista                   0.36.1             pyhd8ed1ab_0    conda-forge
...
vtk                       9.0.3           py38h84a6d72_201
```

We can see that PyVista was pulled from the conda-forge channel (expected) but VTK was not... not sure where it came from. I recommend making sure that VTK is installed from the conda-forge channel. To do so, you may have to uninstall VTK then reinstall it like so:

```py
conda uninstall vtk
conda install -c conda-forge vtk
```

**Reply by earcmag** [reply]
Thanks! Managed to get this to work when I created a new environment with everything in a conda-forge channel
